### PR TITLE
updated: added storage version and managing tags

### PR DESCRIPTION
**Pull request by Akhilesh53**
A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- 0chain: 
- blobber:
- gosdk:
- system_test:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: cmd/newallocation.go [hunk 1]
```diff
@@ -237,6 +237,22 @@ var newallocationCmd = &cobra.Command{
...
 
+		var storageVersion int64
+		if flags.Changed("storage_version") {
+			storageVersion, err = flags.GetInt64("storage_version")
+			if err != nil {
+				log.Fatal("invalid forbid_upload: ", err)
+			}
+		}
+
+		var managingWallet string
```
Comment (Jayashsatolia403) [inline cmd/newallocation.go]: Why did you add managingWallet in newAllocationRequest?

File: cmd/newallocation.go [hunk 2]
```diff
@@ -258,6 +274,8 @@ var newallocationCmd = &cobra.Command{
 				ThirdPartyExtendable: thirdPartyExtendable,
 				Force:                force,
 				IsEnterprise:         isEnterprise,
+				StorageVersion:       storageVersion,
+				ManagingWallet:       managingWallet,
```
Comment (Jayashsatolia403) [inline cmd/newallocation.go]: There is no option in CreateAllocationOptions for Managing wallet.

File: cmd/newallocation.go [hunk 3]
```diff
@@ -355,6 +373,10 @@ func init() {
 	newallocationCmd.Flags().Bool("forbid_move", false, "(default false) specify if the users cannot move objects from this allocation")
 	newallocationCmd.Flags().Bool("forbid_copy", false, "(default false) specify if the users cannot copy object from this allocation")
 	newallocationCmd.Flags().Bool("forbid_rename", false, "(default false) specify if the users cannot rename objects in this allocation")
+
+	newallocationCmd.Flags().String("storage_version", "", "storage version of the blobber")
+	newallocationCmd.Flags().String("managing_wallet", "", "managing wallet of the blobber")
```
Comment (Jayashsatolia403) [inline cmd/newallocation.go]: Remove it

File: cmd/storage.go [hunk 4]
```diff
@@ -290,6 +299,22 @@ var blobberUpdateCmd = &cobra.Command{
...
 
+		var storageVersion int64
+		if flags.Changed("storage_version"){
+			if storageVersion, err = flags.GetInt64("storage_version"); err != nil{
+				log.Fatal(err)
+			}
+			updateBlobber.StorageVersion = &storageVersion
+		}
+
+		var managingWallet string
```
Comment (Jayashsatolia403) [inline cmd/storage.go]: Remove managing wallet, you can't update it.

File: cmd/storage.go [hunk 5]
```diff
@@ -407,6 +432,10 @@ func init() {
 	buf.Bool("not_available", true, "(default false) set blobber's availability for new allocations")
 	buf.Bool("is_restricted", true, "(default false) set is_restricted")
 	buf.String("url", "", "update the url of the blobber, optional")
+	buf.Int64("storage_version", 2, "storage version of the blobber, optional")
+	buf.String("delegate_wallet", "", "delegate wallet of the blobber, optional")
+	buf.String("managing_wallet", "", "managing wallet of the blobber, optional")
```
Comment (Jayashsatolia403) [inline cmd/storage.go]: Remove it.

**Review by Jayashsatolia403 — CHANGES_REQUESTED** [review]

File: cmd/storage.go (hunk 4 above)
Comment (Akhilesh53) [reply cmd/storage.go]: Removed

File: cmd/storage.go (hunk 5 above)
Comment (Akhilesh53) [reply cmd/storage.go]: Removed

File: cmd/newallocation.go (hunk 2 above)
Comment (Akhilesh53) [reply cmd/newallocation.go]: Removed. Got confused bw Delegate and Managing wallet

File: cmd/newallocation.go (hunk 3 above)
Comment (Akhilesh53) [reply cmd/newallocation.go]: Removed

File: cmd/newallocation.go (hunk 1 above)
Comment (Akhilesh53) [reply cmd/newallocation.go]: Removed
